bi_sql_editor/views/view_bi_sql_view.xml

```diff
@@ -178,6 +178,10 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                                         name="is_group_by"
                                         attrs="{'invisible': [('field_description', '=', False)]}"
                                     />
+                                    <field
+                                        name="is_many2one_clickable"
```

Comment:
I would add this after many2one_model_id, as is related to many2one too.

Reply:
I have to say that sometimes you are right, and this is one of those few times. thank you for the input 🙂